common/src/main/java/tachyon/util/io/BufferUtils.java

```diff
@@ -362,5 +362,17 @@ public static ByteBuffer sliceByteBuffer(ByteBuffer buffer, int position, int le
     return slicedBuffer;
   }
 
+  /**
+   * Convenience method for {@link #sliceByteBuffer(ByteBuffer, int, int)} where the last parameter
+   * is the number of remaining bytes in the new buffer.
+   *
+   * @param buffer source {@link ByteBuffer} to slice
+   * @param position position in the source {@link ByteBuffer} to slice
+   * @return the sliced {@link ByteBuffer}
+   */
+  public static ByteBuffer sliceByteBuffer(ByteBuffer buffer, int position) {
+    return ((ByteBuffer) buffer.duplicate().position(position)).slice();
```

Comment:
since this is a convenience method, should it call `#sliceByteBuffer(buffer, position, buffer.remaining())`? I'd expect the `duplicate` to dwarf the cost of `limit` and `remaining`